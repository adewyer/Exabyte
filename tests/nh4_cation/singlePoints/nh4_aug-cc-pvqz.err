rtdb_seq_get: type mismatch "bgj:scf_type" in ./nh4_aug-cc-pvqz.db: arg=1010, db=0
0:bgj_get_scf_method: error reading rtdb:Received an Error in Communication
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI COMMUNICATOR 3 DUP FROM 0
with errorcode -1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
