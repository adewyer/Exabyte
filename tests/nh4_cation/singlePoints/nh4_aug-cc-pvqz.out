 argument  1 = nh4_aug-cc-pvqz.inp
                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.0.0
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2019
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = Amanda-Dewyers-MacBook-Air.local
    program         = nwchem
    date            = Fri Sep 11 17:34:12 2020

    compiled        = Fri_Jul_03_06:05:09_2020
    source          = /private/tmp/nwchem-20200703-71421-tc8c0i/nwchem-7.0.0
    nwchem branch   = 7.0.0
    nwchem revision = 2c9a1c7c
    ga revision     = 5.7.1
    use scalapack   = T
    input           = nh4_aug-cc-pvqz.inp
    prefix          = nh4_aug-cc-pvqz.
    data base       = ./nh4_aug-cc-pvqz.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                     test case
                                     ---------
 C1  symmetry detected

          ------
          auto-z
          ------
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 N                    7.0000     0.00001682    -0.00000000     0.00000806
    2 H                    1.0000    -0.99011928    -0.20294100     0.17987908
    3 H                    1.0000     0.56649954    -0.82246279     0.23776620
    4 H                    1.0000     0.29585203     0.79730895     0.57506355
    5 H                    1.0000     0.12764994     0.22809484    -0.99276521

      Atomic Mass 
      ----------- 

      N                 14.003070
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)      30.8536052699

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000    -0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.54325
    2 Stretch                  1     3                       0.54325
    3 Stretch                  1     4                       0.54325
    4 Stretch                  1     5                       0.54326
    5 Stretch                  2     3                       0.88710
    6 Stretch                  2     4                       0.88712
    7 Stretch                  2     5                       0.88711
    8 Stretch                  3     4                       0.88717
    9 Stretch                  3     5                       0.88714
   10 Stretch                  4     5                       0.88712
   11 Bend                     1     2     3                35.26638
   12 Bend                     1     2     4                35.26482
   13 Bend                     1     2     5                35.26646
   14 Bend                     1     3     2                35.26699
   15 Bend                     1     3     4                35.26038
   16 Bend                     1     3     5                35.26365
   17 Bend                     1     4     2                35.26481
   18 Bend                     1     4     3                35.25976
   19 Bend                     1     4     5                35.26518
   20 Bend                     1     5     2                35.26630
   21 Bend                     1     5     3                35.26289
   22 Bend                     1     5     4                35.26504
   23 Bend                     2     1     3               109.46663
   24 Bend                     2     1     4               109.47036
   25 Bend                     2     1     5               109.46724
   26 Bend                     2     3     4                59.99933
   27 Bend                     2     3     5                59.99942
   28 Bend                     2     4     3                59.99628
   29 Bend                     2     4     5                59.99890
   30 Bend                     2     5     3                59.99811
   31 Bend                     2     5     4                60.00063
   32 Bend                     3     1     4               109.47985
   33 Bend                     3     1     5               109.47345
   34 Bend                     3     2     4                60.00440
   35 Bend                     3     2     5                60.00248
   36 Bend                     3     4     5                59.99925
   37 Bend                     3     5     4                60.00299
   38 Bend                     4     1     5               109.46979
   39 Bend                     4     2     5                60.00047
   40 Bend                     4     3     5                59.99776
   41 Torsion                  1     2     3     4          35.26008
   42 Torsion                  1     2     3     5         -35.26647
   43 Torsion                  1     2     4     3         -35.26164
   44 Torsion                  1     2     4     5          35.26819
   45 Torsion                  1     2     5     3          35.26640
   46 Torsion                  1     2     5     4         -35.26656
   47 Torsion                  1     3     2     4         -35.26008
   48 Torsion                  1     3     2     5          35.26647
   49 Torsion                  1     3     4     2          35.26668
   50 Torsion                  1     3     4     5         -35.26257
   51 Torsion                  1     3     5     2         -35.26981
   52 Torsion                  1     3     5     4          35.25930
   53 Torsion                  1     4     2     3          35.26164
   54 Torsion                  1     4     2     5         -35.26819
   55 Torsion                  1     4     3     2         -35.26668
   56 Torsion                  1     4     3     5          35.26257
   57 Torsion                  1     4     5     2          35.26782
   58 Torsion                  1     4     5     3         -35.25716
   59 Torsion                  1     5     2     3         -35.26640
   60 Torsion                  1     5     2     4          35.26656
   61 Torsion                  1     5     3     2          35.26981
   62 Torsion                  1     5     3     4         -35.25930
   63 Torsion                  1     5     4     2         -35.26782
   64 Torsion                  1     5     4     3          35.25716
   65 Torsion                  2     1     3     4        -120.00142
   66 Torsion                  2     1     3     5         119.99368
   67 Torsion                  2     1     4     3         119.99913
   68 Torsion                  2     1     4     5        -119.99373
   69 Torsion                  2     1     5     3        -119.99330
   70 Torsion                  2     1     5     4         119.99564
   71 Torsion                  2     3     1     4         120.00142
   72 Torsion                  2     3     1     5        -119.99368
   73 Torsion                  2     3     4     5         -70.52925
   74 Torsion                  2     3     5     4          70.52911
   75 Torsion                  2     4     1     3        -119.99913
   76 Torsion                  2     4     1     5         119.99373
   77 Torsion                  2     4     3     5          70.52925
   78 Torsion                  2     4     5     3         -70.52498
   79 Torsion                  2     5     1     3         119.99330
   80 Torsion                  2     5     1     4        -119.99564
   81 Torsion                  2     5     3     4         -70.52911
   82 Torsion                  2     5     4     3          70.52498
   83 Torsion                  3     1     2     4         120.00723
   84 Torsion                  3     1     2     5        -119.99748
   85 Torsion                  3     1     4     5         120.00714
   86 Torsion                  3     1     5     4        -120.01106
   87 Torsion                  3     2     1     4        -120.00723
   88 Torsion                  3     2     1     5         119.99748
   89 Torsion                  3     2     4     5          70.52983
   90 Torsion                  3     2     5     4         -70.53296
   91 Torsion                  3     4     1     5        -120.00714
   92 Torsion                  3     4     2     5         -70.52983
   93 Torsion                  3     5     1     4         120.01106
   94 Torsion                  3     5     2     4          70.53296
   95 Torsion                  4     1     2     5         119.99528
   96 Torsion                  4     1     3     5        -120.00490
   97 Torsion                  4     2     1     5        -119.99528
   98 Torsion                  4     2     3     5         -70.52655
   99 Torsion                  4     3     1     5         120.00490
  100 Torsion                  4     3     2     5          70.52655


            XYZ format geometry
            -------------------
     5
 geometry
 N                     0.00000890    -0.00000000     0.00000426
 H                    -0.52394860    -0.10739176     0.09518792
 H                     0.29977867    -0.43522860     0.12582046
 H                     0.15655816     0.42191776     0.30431055
 H                     0.06754944     0.12070260    -0.52534876

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 N                |     1.02660  |     1.02660
    3 H                |   1 N                |     1.02658  |     1.02658
    4 H                |   1 N                |     1.02660  |     1.02660
    5 H                |   1 N                |     1.02660  |     1.02660
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          4
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 N                |   3 H                |   109.47
    2 H                |   1 N                |   4 H                |   109.47
    2 H                |   1 N                |   5 H                |   109.47
    3 H                |   1 N                |   4 H                |   109.48
    3 H                |   1 N                |   5 H                |   109.47
    4 H                |   1 N                |   5 H                |   109.47
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          6
 ==============================================================================



  library name resolved from: .nwchemrc
  library file name is: </usr/local/Cellar/nwchem/7.0.0_2/share/nwchem/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                        aug-cc-pvqz                 on all atoms 



                                 NWChem DFT Module
                                 -----------------


                                     test case


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  4.58400000E+04  0.000092
  1 S  6.86800000E+03  0.000717
  1 S  1.56300000E+03  0.003749
  1 S  4.42400000E+02  0.015532
  1 S  1.44300000E+02  0.053146
  1 S  5.21800000E+01  0.146787
  1 S  2.03400000E+01  0.304663
  1 S  8.38100000E+00  0.397684
  1 S  3.52900000E+00  0.217641

  2 S  4.58400000E+04 -0.000020
  2 S  6.86800000E+03 -0.000159
  2 S  1.56300000E+03 -0.000824
  2 S  4.42400000E+02 -0.003478
  2 S  1.44300000E+02 -0.011966
  2 S  5.21800000E+01 -0.035388
  2 S  2.03400000E+01 -0.080077
  2 S  8.38100000E+00 -0.146722
  2 S  3.52900000E+00 -0.116360

  3 S  1.05400000E+00  1.000000

  4 S  4.11800000E-01  1.000000

  5 S  1.55200000E-01  1.000000

  6 S  5.46400000E-02  1.000000

  7 P  4.93300000E+01  0.005533
  7 P  1.13700000E+01  0.037962
  7 P  3.43500000E+00  0.149028

  8 P  1.18200000E+00  1.000000

  9 P  4.17300000E-01  1.000000

 10 P  1.42800000E-01  1.000000

 11 P  4.40200000E-02  1.000000

 12 D  2.83700000E+00  1.000000

 13 D  9.68000000E-01  1.000000

 14 D  3.35000000E-01  1.000000

 15 D  1.11000000E-01  1.000000

 16 F  2.02700000E+00  1.000000

 17 F  6.85000000E-01  1.000000

 18 F  2.45000000E-01  1.000000

 19 G  1.42700000E+00  1.000000

 20 G  5.59000000E-01  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  8.26400000E+01  0.002006
  1 S  1.24100000E+01  0.015343
  1 S  2.82400000E+00  0.075579

  2 S  7.97700000E-01  1.000000

  3 S  2.58100000E-01  1.000000

  4 S  8.98900000E-02  1.000000

  5 S  2.36300000E-02  1.000000

  6 P  2.29200000E+00  1.000000

  7 P  8.38000000E-01  1.000000

  8 P  2.92000000E-01  1.000000

  9 P  8.48000000E-02  1.000000

 10 D  2.06200000E+00  1.000000

 11 D  6.62000000E-01  1.000000

 12 D  1.90000000E-01  1.000000

 13 F  1.39700000E+00  1.000000

 14 F  3.60000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 N                        aug-cc-pvqz               20      105   6s5p4d3f2g
 H                        aug-cc-pvqz               14       55   5s4p3d2f




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 N                        aug-cc-pvqz               20      105   6s5p4d3f2g
 H                        aug-cc-pvqz               14       55   5s4p3d2f


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     5
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     1
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:   325
                     number of shells:    76
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                         B3LYP Method XC Potential
                     Hartree-Fock (Exact) Exchange  0.200          
                        Slater Exchange Functional  0.800 local    
                    Becke 1988 Exchange Functional  0.720 non-local
              Lee-Yang-Parr Correlation Functional  0.810          
                  VWN I RPA Correlation Functional  0.190 local    

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          N                   0.65       49          10.0       434
          H                   0.35       45          11.0       434
          Grid pruning is: on 
          Number of quadrature shells:   229
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


 WARNING : Found    31 linear dependencies 
 S eigenvalue threshold:     1.00000E-05
 Smallest S eigenvalue :     1.72466E-08
 Largest  S eigenvalue :     9.13490E-06


 !! The overlap matrix has  31 vectors deemed linearly dependent with
    eigenvalues:
 1.72D-08 3.17D-08 4.60D-08 2.28D-07 2.30D-07 2.31D-07 2.37D-07 3.79D-07
 3.80D-07 3.81D-07 1.11D-06 1.21D-06 1.36D-06 1.43D-06 1.67D-06 2.31D-06
 2.43D-06 2.44D-06 2.45D-06 2.96D-06 3.25D-06 3.57D-06 3.68D-06 4.10D-06
 4.44D-06 4.66D-06 6.30D-06 6.95D-06 7.35D-06 7.71D-06 9.13D-06


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -56.31251883

 Renormalizing density from      11.00 to     10

      Non-variational initial energy
      ------------------------------

 Total energy =     -52.442120
 1-e energy   =    -113.943503
 2-e energy   =      30.647778
 HOMO         =      -2.013693
 LUMO         =      -0.295505

   Time after variat. SCF:     99.7
   Time prior to 1st pass:     99.7

 Integral file          = ./nh4_aug-cc-pvqz.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  32766
 Max. records in memory =      0        Max. records in file   = 141640
 No. of bits per label  =     16        No. of bits per value  =     64


 #quartets = 4.282D+06 #integrals = 1.437D+09 #direct =  0.0% #cached =100.0%

 ------------------------------------------------------------------------
 bgj_get_scf_method: error reading rtdb                   0
 ------------------------------------------------------------------------
 ------------------------------------------------------------------------
  current input line : 
    19: task dft energy
 ------------------------------------------------------------------------
 ------------------------------------------------------------------------
 An error occured in the Runtime Database
 ------------------------------------------------------------------------
 For more information see the NWChem manual at https://github.com/nwchemgit/nwchem/wiki


 For further details see manual section:                                                                                                                                                                                                                                                                
